iwyu_tool.py

```diff
@@ -53,16 +54,17 @@
 GENERAL, ADD, REMOVE, LIST = range(4)
 
 
-def clang_formatter(output, style):
+def clang_formatter(output, style, logLevel):
```

Comment:
Naming style is mixed across different python scripts, but newer ones like `iwyu_tool.py` use `hacker_case_with_underscores`.